### DISCUSSION TITLE
Speedrun.com page

### DISCUSSION THREAD

**Discussion by nekoify** [aeolus-1/triangleGame.github.io#54] [Ideas]
we should make Speedrun.com page

**Comment by serifonium** [comment]
yeah

**Comment by aeolus-1** [comment]
You would have to have footage from reload to end for this game

**Reply by aeolus-1** [reply]
also I think two people have completed the game without cheating

**Reply by nekoify** [reply]
there could be diff categories like completing the game legitly but also things like using inf jump

**Reply by Quantaviousdingleberry420** [reply]
yeah do no cheats any% and cheats any%
